packages/dd-trace/src/appsec/rule_manager.js

```diff
@@ -95,6 +90,7 @@ function updateWafFromRC ({ toUnapply, toApply, toModify }) {
       }
 
       if (file && file.actions && file.actions.length) {
+        batchConfiguration = true
         newActions.set(id, file.actions)
       }
 
```

Comment:
in the line just below, it says:
>"actions" data is managed by tracer and not by waf

It's not anymore right ? so we can revert the whole batchConfiguration thing to how it was before it was implemented ?